Source/Dafny/Dafny.csproj

```diff
@@ -0,0 +1,36 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+      <OutputType>Exe</OutputType>
+      <AssemblyName>Dafny</AssemblyName>
+      <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+      <DefineConstants>TRACE</DefineConstants>
+      <TargetFramework>net6.0</TargetFramework>
+      <OutputPath>..\..\Binaries\</OutputPath>
+      <ValidateExecutableReferencesMatchSelfContained>false</ValidateExecutableReferencesMatchSelfContained>
+      <VersionPrefix>3.8.1.40901</VersionPrefix>
+
+      <PackAsTool>true</PackAsTool>
+      <ToolCommandName>dafny</ToolCommandName>
+      <PackageLicenseExpression>MIT</PackageLicenseExpression>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Debug|net6.0|AnyCPU'">
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+    <AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
+  </PropertyGroup>
+
+  <!-- Working around some stange behavior in dotnet publish: https://github.com/dotnet/sdk/issues/10566 -->
+  <PropertyGroup Condition="$(RUNTIME_IDENTIFIER) != ''">
+    <RuntimeIdentifier>$(RUNTIME_IDENTIFIER)</RuntimeIdentifier>
+  </PropertyGroup>
+    
+  <ItemGroup>
+    <ProjectReference Include="..\DafnyDriver\DafnyDriver.csproj" />
+    <ProjectReference Include="..\DafnyPipeline\DafnyPipeline.csproj" />
```

Comment:
Does this need `DafnyPipeline`?

Reply:
Absolutely, since that's how the runtime files are included. I understand why you ask, because the minimal contents of `Dafny` should just be `DafnyDriver + runtimes`, but `DafnyPipeline` is exactly the assembly that embeds the runtimes. Most importantly the compilers load them from the `DafnyPipeline.dll` assembly explicitly rather than whatever the currently executing assembly is.

Reply:
Right, of course!